README.md

```diff
@@ -1,3 +1,4 @@
+[![Build Status](https://travis-ci.org/Harvard-ATG/visualizing_russian_tools.svg?branch=master)](https://travis-ci.org/Harvard-ATG/visualizing_russian_tools)
```

Comment:
This looks like an old version of the `README.md` that's trying to be merged in. Do you have the latest version in your repo?